.editorconfig

```diff
@@ -0,0 +1,447 @@
+
+dotnet_naming_rule.const_field_members_should_be_pascal_case.severity = error
+dotnet_naming_rule.const_field_members_should_be_pascal_case.symbols = const_field_members
+dotnet_naming_rule.const_field_members_should_be_pascal_case.style = pascal_case
+
+dotnet_naming_rule.local_var_and_method_args.severity = error
+dotnet_naming_rule.local_var_and_method_args.symbols = local_var_and_method_args
+dotnet_naming_rule.local_var_and_method_args.style = camel_case
+
+dotnet_naming_rule.private_member_variables.severity = error
+dotnet_naming_rule.private_member_variables.symbols = private_member_variables
+dotnet_naming_rule.private_member_variables.style = pascal_case_with_underscore_prefix
+
+# Symbol specifications
+
+dotnet_naming_symbols.interface.applicable_kinds = interface
+dotnet_naming_symbols.interface.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.interface.required_modifiers =
+
+dotnet_naming_symbols.const_field_members.applicable_kinds = field
+dotnet_naming_symbols.const_field_members.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.const_field_members.required_modifiers = const
+
+dotnet_naming_symbols.types.applicable_kinds = class, struct, interface, enum
+dotnet_naming_symbols.types.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.types.required_modifiers =
+
+dotnet_naming_symbols.non_field_members.applicable_kinds = property, event, method
+dotnet_naming_symbols.non_field_members.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.non_field_members.required_modifiers =
+
+dotnet_naming_symbols.private_member_variables.applicable_kinds = field
+dotnet_naming_symbols.private_member_variables.applicable_accessibilities = private, private_protected
+dotnet_naming_symbols.private_member_variables.required_modifiers =
+
+dotnet_naming_symbols.local_var_and_method_args.applicable_kinds = parameter, local
+
+# Naming styles
+
+dotnet_naming_style.begins_with_i.required_prefix = I
+dotnet_naming_style.begins_with_i.required_suffix =
+dotnet_naming_style.begins_with_i.word_separator =
+dotnet_naming_style.begins_with_i.capitalization = pascal_case
+
+dotnet_naming_style.pascal_case.required_prefix =
+dotnet_naming_style.pascal_case.required_suffix =
+dotnet_naming_style.pascal_case.word_separator =
+dotnet_naming_style.pascal_case.capitalization = pascal_case
+
+dotnet_naming_style.pascal_case_with_underscore_prefix.required_prefix = _
+dotnet_naming_style.pascal_case_with_underscore_prefix.required_suffix =
+dotnet_naming_style.pascal_case_with_underscore_prefix.word_separator =
+dotnet_naming_style.pascal_case_with_underscore_prefix.capitalization = pascal_case
+

```

Comment:
I see, I think you have to define a custom naming style here: `camel_case_with_underscore_prefix` (basically the same as `pascal_case_with_underscore_prefix` but with the capitalization changed)